application/models/entitycode.go

```diff
@@ -32,6 +32,16 @@ func (ec *EntityCode) Create(tx *pop.Connection) error {
 	return create(tx, ec)
 }
 
+//create a new entity code from an api entity code
+func (ec *EntityCode) CreateFromAPI(tx *pop.Connection, input api.EntityCodeInput) error {
+	ec.Code = input.Code
```

Comment:
```suggestion
	ec.ID = EntityCodeID(input.Code)
	ec.Code = input.Code
```
This is to create a V5 UUID instead of letting the ORM library, Pop, create a V4 UUID.